e2e-execute.sh

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+shadowJar() {
+  echo "Running shadowJar..."
+  ./gradlew shadowJar
+}
+
 start_api() {
+    DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+    SUB_DIR="app/build/libs"
+    JAR_NAME="app-all.jar"
     echo 'Starting API'
-    ./gradlew --no-daemon app:clean app:run &
+    java -jar "${DIR}"/"${SUB_DIR}"/"${JAR_NAME}" > /dev/null &
     export API_PID="${!}"
     echo "API starting at PID ${API_PID}"
 }
 
 wait_for_api() {
     attempt_counter=0
-    max_attempts=36
+    max_attempts=20
```

Comment:
I brought down the `max_attempts` to 20. I think it is more than enough attempts in order to check if the api is running. There is a 5 milliseconds interval per attempt. Let me know your thoughts if I should keep it at 36 attempts.

Reply:
It's actually 5 seconds.  But 1.6 minutes instead of 3 minutes should be fine.